lib/sparkpost.js

```diff
@@ -106,6 +108,9 @@ SparkPost.prototype.request = function(options, callback) {
     options.gzip = true;
   }
 
+  // set debug
+  options.debug = (typeof options.debug === 'boolean') ? options.debug : this.debug;
```

Comment:
oh... i didn't get what this was doing... right, so this line is setting the override for a single call, defaulting to the client config. I got it now 😄